kernel/scripts/mkdisk_img.sh

```diff
@@ -109,7 +109,7 @@ set menu_color_highlight="light-green/black"
 # TempOS
 #
```

Comment:
guess we can make another menu entry, for instance, "TempOS" and "TempOS over serial", something like that...